### PR TITLE
add some test cases for util package

### PR DESCRIPTION
**Pull request by javeme**
Change-Id: I39b457c2a7a338a909b3aa0ba8ff4a8b5057b339

### CODE REVIEW

**Comment by javeme** [comment]
Before adding test cases:
image

**Comment by javeme** [comment]
After adding test cases:
image

**Review by Linary — CHANGES_REQUESTED** [review]

File: src/main/java/com/baidu/hugegraph/util/CollectionUtil.java [hunk 1]
```diff
@@ -108,10 +108,14 @@ public static boolean allUnique(Collection<?> collection) {
      * @return sub-set of original set [from, to)
      */
     public static <T> Set<T> subSet(Set<T> original, int from, int to) {
-        List<T> list = new ArrayList<>(original);
-        if (to == -1) {
+        if (from < 0) {
+            E.checkArgument(from >= 0,
```
[the remark this answers is not in the document]
Comment (javeme) [reply src/main/java/com/baidu/hugegraph/util/CollectionUtil.java:112]: to keep same style with "if (to < 0)"

**Review by corgiboygsj — APPROVED** [review]

File: src/main/java/com/baidu/hugegraph/util/CollectionUtil.java [hunk 2]
```diff
@@ -108,10 +114,12 @@ public static boolean allUnique(Collection<?> collection) {
      * @return sub-set of original set [from, to)
      */
     public static <T> Set<T> subSet(Set<T> original, int from, int to) {
-        List<T> list = new ArrayList<>(original);
-        if (to == -1) {
+        E.checkArgument(from >= 0,
+                        "Invalid from parameter of subSet(): %s", from);
+        if (to < 0) {
```
Comment (houzhizhen) [inline src/main/java/com/baidu/hugegraph/util/CollectionUtil.java:119]: "if (to < 0)" => "if (to < from)". IllegalArgumentException will be thrown if  to < from。

**Review by houzhizhen — APPROVED** [review]

**Review by Linary — APPROVED** [review]

**Pull request merged**